### ISSUE TITLE
Pikaday

### ISSUE THREAD

**Issue by rakarachon** [IIC2513-2019-1/syllabus#32]
Hola! Podemos usar [pikaday](https://github.com/Pikaday/Pikaday) para la selección de fechas en los forms?

**Comment by gpvidal** [comment]
Sí, podrían agregarlo. Ojo que la integración podría ser complicada.

Saludos,
